tests/config/simple-arpa.yaml

```diff
@@ -1,6 +1,7 @@
 manager:
   max_workers: 2
   auto_arpa:
+    populate_should_replace: True
```

Comment:
Any specific reasons for adding this and checking it in the tests. It seems fine just wondering what lead to the change as it doesn't seem to be related to coverage. Just testing that the param was passed through correctly?

Reply:
Thanks for the quick release. Yes, I added that to test if the param is actually making its way through correctly.